cmd/node/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"os"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/node-monitoring/config"
+	"github.com/ElrondNetwork/node-monitoring/monitoring"
+	"github.com/urfave/cli"
+)
+
+var (
+	cliHelpTemplate = `NAME:
+   {{.Name}} - {{.Usage}}
+USAGE:
+   {{.HelpName}} {{if .VisibleFlags}}[global options]{{end}}
+   {{if len .Authors}}
+AUTHOR:
+   {{range .Authors}}{{ . }}{{end}}
+   {{end}}{{if .Commands}}
+GLOBAL OPTIONS:
+   {{range .VisibleFlags}}{{.}}
+   {{end}}
+VERSION:
+   {{.Version}}
+   {{end}}
+`
+	log = logger.GetOrCreate("eventNotifier")
+
+	generalConfigFile = cli.StringFlag{
+		Name:  "general-config",
+		Usage: "The path for the general config",
+		Value: "./config/config.toml",
+	}
+)
+
+func main() {
+	app := cli.NewApp()
+	cli.AppHelpTemplate = cliHelpTemplate
+	app.Name = "Elrond Node Monitoring"
+	app.Flags = []cli.Flag{
+		generalConfigFile,
+	}
+	app.Authors = []cli.Author{
+		{
+			Name:  "The Elrond Team",
+			Email: "contact@elrond.com",
+		},
+	}
+	app.Action = startNodeMonitoring
+
+	err := app.Run(os.Args)
+	if err != nil {
+		log.Error(err.Error())
+		os.Exit(1)
+	}
+}
+
+func startNodeMonitoring(ctx *cli.Context) error {
+	log.Info("starting node monitoring tool...")
+
+	flagsConfig, err := getFlagsConfig(ctx)
+	if err != nil {
+		return err
+	}
+
+	cfg, err := config.LoadConfig(flagsConfig.GeneralConfigPath)
+	if err != nil {
+		return err
+	}
+	cfg.Flags = flagsConfig
+
+	runner, err := monitoring.NewMonitoringRunner(cfg)
+	if err != nil {
+		return err
+	}
+
+	err = runner.Start()
```

Comment:
how does the application stop if we hit CTRL+C? suggestion to implement the hook to the SIGINT/SIGTERM os signals, as seen here: https://github.com/ElrondNetwork/elrond-go/blob/b7668c2e4e8468cd9c13abf2a06a3f4c0139deb2/node/nodeRunner.go#L463

Reply:
:+1: